scenes/game/game.gd

```diff
@@ -2518,10 +2549,23 @@ func _on_life_damage_dealt(event_data):
 	# Example: `Deal 1 life damage`
 	_play_popup_message(tr("DEAL_LIFE_DAMAGE").format({Amount=life_lost}))
 
-	var game_over = event_data["game_over"]
-	if game_over:
+	if event_data["game_over"]:
 		target_player.life_count -= life_lost
 
+func _on_attached_action_activation(event_data):
+	var active_player = get_player(event_data["event_player_id"])
+	var effect_id = event_data["effect_id"]
+	var support_id = event_data["support_id"]
+	var logline = "%s Support Card Effect [SKILL]%s[/SKILL] activated: [CARD]%s[/CARD]" % [
```

Comment:
not sure how you might want to change this to account for special actions.
Maybe just having this say "Special Action: skill activated"